doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -445,16 +429,23 @@ Requests to the environment
       by_notation ::= @string {? % @scope_key }
 
    Displays the full name of objects from |Coq|'s various qualified namespaces such as terms,
-   modules and Ltac.  It also displays notation definitions.
+   modules and Ltac, thereby showing the module they are defined in.  It also displays notation definitions.
 
-   If the argument is:
-
-   * :n:`@qualid` - displays the full name of objects that
-     end with :n:`@qualid`, thereby showing the module they are defined in.
-   * :n:`@string {? "%" @ident }` - displays the definition of a notation.  :n:`@string`
+   * :n:`@qualid` - refers to object names that end with :n:`@qualid`.
+   * :n:`@string {? % @scope_key }` - refers to definitions of notations.  :n:`@string`
      can be a single token in the notation such as "`->`" or a pattern that matches the
      notation.  See :ref:`locating-notations`.
 
+     :n:`% @scope_key`, if present, limits the reference to the scope bound to the delimiting
+     key :n:`@scope_key`, such as, for example, :n:`%nat`.  (see Section
+     :ref:`LocalInterpretationRulesForNotations`)
+
+     .. note::
+
+        To refer to a string used in a notation that is a substring of a valid identifier,
+        put it between single quotes or explicitly provide a scope.
+        See :ref:`this example <search-disambiguate-notation>`.
```

Comment:
This note is specific to the case of `Search` where strings are ambiguous (can designate part of identifiers or notations), so it shouldn't have been moved.